pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiActivityValidation.java

```diff
@@ -78,11 +77,6 @@ int _insertActivityValidation(
     List<ActivityValidationDto> _findByActivityId(@Bind("activityId") long activityId);
 
     default int _deleteByActivityId(long activityId) {
-        List<ActivityValidationDto> activityValidations = _findByActivityId(activityId);
-        List<Long> validationIdsForActivity = activityValidations
-                .stream()
-                .map(dto -> dto.getActivityValidationId())
-                .collect(Collectors.toList());
         return _deleteValidationsByActivityId(activityId);
```

Comment:
Noticed that we have a useless selection in the delete method (maybe someone forgot to remove debug stuff).